src/phoenix/server/app.py

```diff
@@ -167,8 +168,11 @@ async def dispatch(
         request: Request,
         call_next: RequestResponseEndpoint,
     ) -> Response:
+        from phoenix import __version__ as phoenix_version
```

Comment:
this can go to top of the file?

Reply:
because we import so much at the top level, that would cause a circular import

Reply:
got it